src/cml.js

```diff
@@ -464,7 +465,7 @@ class CML {
       await exec(`git checkout -b ${source}`);
       await exec(`git add ${paths.join(' ')}`);
       let commitMessage = `CML PR for ${shaShort}`;
-      if (!(merge || rebase || squash)) {
+      if (skipCI || !(merge || rebase || squash)) {
```

Comment:
per the last line in [this comment](https://github.com/iterative/cml/issues/1023#issuecomment-1140154009) I wonder if `skipCI` should mean ` [skip ci]` is added to `title` rather than `commitMessage`?

Reply:
I suppose you refer to `option to add [skip ci] to the PR title (afaik the PR title usually becomes the commit title upon merge)`?

I checked and on GitLab it won't work. The commit message is something like:
"Merge branch 'feature-branch-cml-pr-10d62239' into 'feature-branch''"

The proper way would be to go the way I sketched [here](https://github.com/iterative/cml/issues/1023#issuecomment-1141277204).

Reply:
And btw, as of now, even with the current PR, the issue would still be open. The only way to avoid undesired CI runs on the feature-branch after the merge is to avoid setting auto-merge, and manually add [skip ci] to the commit message before merging the PR created by CML.

Reply:
Right now it looks good.
GH and GL will honor skip ci in the commit message.